packages/slate-react/src/components/editable.tsx

```diff
@@ -1049,6 +1053,37 @@ export const Editable = (props: EditableProps) => {
   )
 }
 
+/**
+ * The props that get passed to renderPlaceholder
+ */
+export type RenderPlaceholderProps = {
+  contentEditable: boolean
+  children?: React.ReactNode
+  ref: React.RefObject<any>
+}
+
+/**
+ * Default placeholder style object.
+ */
+export const defaultPlaceholderStyle: React.CSSProperties = {
+  position: 'absolute',
+  pointerEvents: 'none',
+  width: '100%',
+  maxWidth: '100%',
+  display: 'block',
+  opacity: '0.333',
+  userSelect: 'none',
+  textDecoration: 'none',
+}
```

Comment:
I totally see what you mean now about how these styles are required to get things to work properly, my bad for not understanding before. In that case having it inside `attributes.style` as you did before sounds good to me, and people can cherry-pick if they really need to.

Just since I'd rather not add another export for people to keep track of.